lua/load_wezterm_nvim.lua

```diff
@@ -0,0 +1,97 @@
+-- loads the wezterm.nvim plugin and exposes methods to the python remote plugin
+local ok, wezterm = pcall(require, "wezterm")
+if not ok then
+  vim.api.nvim_err_writeln("[Molten] `wezterm.nvim` not found")
+  return
+end
+
+local wezterm_api = {}
+
+wezterm_api.get_pane_id = function()
+  local current_pane_id = wezterm.get_current_pane()
+  return current_pane_id
+end
+
+--- Validate the split direction
+--- type function
+--- @param direction string the direction to validate
+--- @return string validated direction if valid
+local validate_split_dir = function(direction)
+  local accepted_dirs = { "top", "bottom", "left", "right" }
+  --if direction not in accepted_dirs, return "bottom" else return direction
+  if not vim.tbl_contains(accepted_dirs, direction) then
+    vim.api.nvim_err_writeln(
+      "'molten_split_dir' must be one of 'top', 'bottom', 'left', or 'right', defaulting to 'right'"
+    )
+    return "right"
+  end
+  return direction
+end
+
+--- Validate the split size
+--- type function
+--- @param size number the size to validate
+--- @return number validated size if valid
+local validate_split_size = function(size)
+  if size < 0 or size > 100 or size == nil then
+    vim.api.nvim_err_writeln(

```

Comment:
Can you use the notify api here? I think that I'd prefer if this was just a warning, b/c we do have a validation and default that we fall back to.

Reply:
Do you want me to create a separate lua module that has the wrapper functions setup for the notify api like you did on the python side...or can we use pynvim to call that python code from this side (which seems maybe weird because its calling right back to the vim side of things?). I looked at the load_image_nvim.lua for reference initially and didn't see the notify usuage so that is why I chose this route.

Reply:
@benlubas just wanted to tag you in this question because I know git comments get lost in all the notifications for me sometimes.

Reply:
Sorry, even that ping got lost. You can just use the notify API. If the lua code base grows we can refactor then. Just do this to make sure the messages are consistent:

```lua
vim.notify("[Molten] <your message>", ...)
```

@akthe-at